diff --git a/node_modules/react-quill/lib/index.js b/node_modules/react-quill/lib/index.js
index b77e914..af0cfcd 100644
--- a/node_modules/react-quill/lib/index.js
+++ b/node_modules/react-quill/lib/index.js
@@ -320,7 +320,13 @@ var ReactQuill = /** @class */ (function (_super) {
         if (!this.editingArea) {
             throw new Error('Instantiating on missing editing area');
         }
-        var element = react_dom_1.default.findDOMNode(this.editingArea);
+        var element = this.editingArea;
+        if (element && element.current) {
+            element = element.current;
+        }
+        if (!element && (react_dom_1 === null || react_dom_1 === void 0 ? void 0 : react_dom_1.default) && typeof react_dom_1.default.findDOMNode === 'function') {
+            element = react_dom_1.default.findDOMNode(this.editingArea);
+        }
         if (!element) {
             throw new Error('Cannot find element for editing area');
         }
